<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Course Advisory System</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>596</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="startAnalyzebut">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>START ANALYZE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>511</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;aakar&quot;;
color: rgb(246, 245, 244);</string>
    </property>
    <property name="text">
     <string>Course Advisory System</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/images/images/mainWindowBg.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileConfirmed">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>206</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;aakar&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>No file chosen ...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>label_2</zorder>
   <zorder>fileConfirmed</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="fileOpen"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
   </widget>
   <widget class="QMenu" name="menurefresh">
    <property name="title">
     <string>refresh</string>
    </property>
    <addaction name="actionrefresh_file"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuhelp"/>
   <addaction name="menuabout"/>
   <addaction name="menurefresh"/>
  </widget>
  <action name="fileOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionrefresh_file">
   <property name="text">
    <string>refresh file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
